decidim-core/app/cells/decidim/author/endorsements.erb

```diff
@@ -1,4 +1,4 @@
 <%= link_to "#{resource_locator(from_context).path}#list-of-endorsements", title: t("decidim.author.endorsements", count: from_context.endorsements_count), class: "author__metadata" do %>
-  <%= icon "heart-add-line" if show_icons? %>
+  <%= icon "like" if show_icons? %>
```

Comment:
Is this change done because it wasn't consistent with other endorsements, right?

Reply:
In the end is the same icon, but, inially we used name parameter. 
This change is actually to avoid defining the same icon twice: 

```
        Decidim.icons.register(name: "like", icon: "heart-add-line", description: "Like", category: "action", engine: :core)
        Decidim.icons.register(name: "heart-add-line", icon: "heart-add-line", category: "system", description: "", engine: :core)
```

Reply:
Maybe it's that I don't understand this, but we should be able to reuse an icon in two places of the application. The logic of the `icon` helper should be: 

- if there's an icon registered with this name, use that.
- if it isn't then:
  - log it on production
  - raise an exception in development or test

Isn't it like that?

Reply:
You got the logic right. 

The small change, done above is to remove any duplicates ( that was a low hanging fruit ) . 
We are using the `like` name in the following files:
```
decidim-core/app/cells/decidim/card_metadata_cell.rb:57:        icon: resource_type_icon_key(:like),
decidim-core/app/cells/decidim/endorsement_buttons_cell.rb:88:      @endorse_icon ||= resource_type_icon(resource.endorsed_by?(current_user) ? "dislike" : "like")
```

and we also have 2 places where  `heart-add-line`

TBH, the change is done only to have a consistent way of displaying things. 
```
decidim-core/app/cells/decidim/author/endorsements.erb:2:  <%= icon "heart-add-line" if show_icons? %>
decidim-proposals/app/views/decidim/proposals/admin/proposals/show.html.erb:135:              <%= icon "heart-add-line", class: "fill-gray" %>

```

Reply:
Ah OK, so, if the answer is "consistency" then you know that you have my full support ✊🏽